crowbar_framework/app/models/node_object.rb

```diff
@@ -170,6 +170,10 @@ def self.all
     self.find nil
   end
 
+  def self.admin_node
+    all.select { |n| n if n.admin? }.first
```

Comment:
Do it like @rsalevsky in his pull request: https://github.com/crowbar/crowbar-core/pull/61/files#diff-496ddf4ffc5694a5fcdddbff8206dc8dR174

```ruby
def self.admin_node
  all.detect(&:admin?)
end
```